spec/models/miq_expression/miq_expression_spec.rb

```diff
@@ -381,6 +381,9 @@
     end
 
     it "should not contain duplicate tag fields" do
+      # tags contain the current user's tenant's name
+      User.current_user = FactoryGirl.create(:user_with_group)
```

Comment:
the model details uses the current tenant's name to modify the label of the tags.
Currently, we don't properly use the current tenants name, but use the default tenant instead. And we currently don't use the tenant model but go directly into the system configuration.

I want to change this so we at least go through the tenant model to determine the tenant name. Per @dclarizio he prefers we keep the original behavior and display the root tenant only (and not the current tenant). I am going to change that just as soon as I get my new dev environment up and running.